File: src/main/java/com/sfh/pokeRogueBot/model/modifier/impl/PokemonNatureChangeModifierItem.java
package com.sfh.pokeRogueBot.model.modifier.impl;

import com.sfh.pokeRogueBot.model.enums.Nature;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class PokemonNatureChangeModifierItem extends PokemonModifierItem {

    public static final String TARGET = "PokemonNatureChangeModifierType";

    private Nature nature;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/modifier/impl/PokemonNatureChangeModifierItem.kt
package com.sfh.pokeRogueBot.model.modifier.impl

import com.sfh.pokeRogueBot.model.enums.Nature

class PokemonNatureChangeModifierItem : PokemonModifierItem() {

    companion object {
        const val TARGET = "PokemonNatureChangeModifierType"
    }

    var nature: Nature? = null
}